h2o-docs/src/product/getting-data-into-h2o.rst

```diff
@@ -43,6 +43,7 @@ Default Data Sources
 - S3 
 - HDFS
 - JDBC
+- Hive
```

Comment:
I don't think you need a Hive bullet here because it is part of JDBC.

Reply:
True, it is accessible through JDBC, however @michalkurka pointed out people may not expect Hive there.

Reply:
Hmm... then perhaps something like:
- S3
- HDFS
- JDBC (including MySQL, PostgreSQL, MariaDB, Netezza, and Hive v2)

Reply:
I would prefer to keep it separate. The reason is that people would not expect it in the same group. It is not a traditional relational db. 

Also - I think the Hive documentation should be extended in the future: our story for Hive is not just JDBC but we also support reading ORC and Parquet directly from HDFS - which is even the preferred approach (in terms of performance).

Reply:
okay